web/src/pages/home/homePage.js

```diff
@@ -617,7 +585,8 @@ const Homepage = () => {
         <div className="caribbean-countries-container">
           {caribbeanCountries.map((country, index) => (
             <div className="country-card caribbean-country-card" key={index}>
-              <a href="https://www.domits.com/home/" target="_blank" rel="noopener noreferrer">
+              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```

Comment:
Dit moet global zijn, niet specifiek naar domits.com
```suggestion
              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```